.github/workflows/scheduled_tests.yml

```diff
@@ -0,0 +1,134 @@
+name: scheduled-tests
+
+on:
+  schedule:
+    - cron:  '0 0 * * *'
+
+jobs:
+
+  linux:
+    name: Tests on Linux
```

Comment:
Why run linux tests on a CRON? We test these in each PR right? Seems like this may be unnecessary to me. If we want a linux test just to compare, maybe we can just run a single test rather than the full matrix?

Reply:
@choldgraf that makes sense. 👍